src/main/java/hng_java_boilerplate/organisation/service/OrganisationService.java

```diff
@@ -0,0 +1,60 @@
+package hng_java_boilerplate.organisation.service;
+
+import hng_java_boilerplate.organisation.entity.Organisation;
+import hng_java_boilerplate.organisation.exception.OrganisationException;
+import hng_java_boilerplate.organisation.exception.response.SuccessResponse;
+import hng_java_boilerplate.organisation.repository.OrganisationRepository;
+import hng_java_boilerplate.user.dto.GetUserDto;
+import hng_java_boilerplate.user.entity.User;
+import hng_java_boilerplate.user.repository.UserRepository;
+import hng_java_boilerplate.user.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.resource.NoResourceFoundException;
+
+import javax.crypto.BadPaddingException;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class OrganisationService {
+    @Autowired
+    private OrganisationRepository organisationRepository;
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserService userService;
+
+    public Organisation getOrganisationDetails(String orgId) throws OrganisationException{
+        Organisation foundOrganisation = organisationRepository.findById(orgId)
+                .orElseThrow(()-> new OrganisationException("Invalid Organisation Id"));
+        return foundOrganisation;
+    }
+
+    public SuccessResponse validateAndAcceptUserToOrganisation(String OrgId){
+//        This is a secured method and while i await Auth guy to come through so i can fetch the id of logged-in user
+// TODO: 7/20/2024 fetch user that wants to be added to organisation from the logged in user
+//        A demo user
```

Comment:
When you signup a user the user is logged in, use that for now

hard coding a user id won't fly

Reply:
please sir. I don't think you checked the latest commit . All these you have mentioned are no longer on my code